### ISSUE TITLE
Improved how history is stored on disk

### ISSUE THREAD

**Issue by Arxcis** [Arxcis/flatstat#9]
**Today**
```json
export default [{
  "appID": "org.gnome.Recipes",
  "ext": "json",
  "branch": "master",
  "stargazerCount": 3,
  "history": [
    {
      "ext": "json",
      "date": "2020-03-12T00:33:28Z",
      "x11": true,
      "fallbackX11": false,
      "wayland": true,
      "filesystem": false,
      "filesystemHome": false,
      "filesystemHost": false,
      "device": false,
      "deviceAll": false,
      "pulseaudio": true,
      "finishArgs": true
    },
```
**Future**
```json
export default [{
  "appID": "org.gnome.Recipes",
  "ext": "json",
  "branch": "master",
  "stargazerCount": 3,
  "history": [
    {
      "ext": "json",
      "date": "2020-03-12T00:33:28Z",
      "finishArgs": {
        "--device": ["dri", "all"],
        "--filesystem": ["̃~/config", "home", "host"],
        "--socket": ["x11", "fallback-x11", "wayland", "pulseaudio"],
       }
    },
```

**Comment by Arxcis** [comment]
**Update testing with com.valve.Steam**
```
  {
    ext: 'yml',
    date: '2020-10-05T12:51:20Z',
    finishArgs: {
      '--allow': [Array],
      '--device': [Array],
      '--extension': [Array],
      '--filesystem': [Array],
      '--persist': [Array],
      '--share': [Array],
      '--socket': [Array],
      '--talk-name': [Array]
    }
```

**Comment by Arxcis** [comment]
fixed by #10
